src/Microsoft.FileFormats/PDB/IMSFFile.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.FileFormats.PDB
+{
+    /// <summary>
+    /// An abstraction for reading both MSF (normal PDB files) and MSFZ files (compressed PDB files).
+    /// </summary>
+    public interface IMSFFile
```

Comment:
You may want to make this internal.  We don't want to expose any more public surface from this assembly unless we absolutely have to.

Reply:
Done.